private/syntax/compile/binding-spec.rkt

```diff
@@ -199,10 +215,27 @@
     [(scope ~! spec ...)
      (scope
       this-syntax
-      (group (map elaborate-bspec (attribute spec))))]
+      (group (elaborate-group (attribute spec))))]
     [(spec ...)
-     (group (map elaborate-bspec (attribute spec)))]))
+     (group (elaborate-group (attribute spec)))]))
 
+; (Listof Syntax) -> (Listof BSpec)
+; handles ellipses
+(define elaborate-group
+  (syntax-parser
+    #:datum-literals (import)
+    [((import v:id (~and ooo/inside (~literal ...)) ...) (~and ooo/outside (~literal ...)) ... . specs)
```

Comment:
I think this approach will fail for something like:

```
#:binding [[(import x) ...] ...]
```

Could we try writing a test for this?